aspnetcore/security/authentication/sociallogins.md

```diff
@@ -159,12 +106,10 @@ To create a password and login using your email that you set during the login pr
 
 ![image](sociallogins/_static/pass2.PNG)
 
-* Set a valid password and you can use this to login with your email
+* Set a valid password and you can use this to login with your email.
 
 ## Next steps
 
-* This article showed how you can authenticate with Facebook. You can follow a similar approach to authenticate with Microsoft Account, Twitter, Google and other providers.
-
-* Once you publish your Web site to Azure Web App, you should reset the AppSecret in the Facebook developer portal.
+* This article introduced external authentication and explained the prerequisites required to add external logins to your ASP.NET Core app.
 
-* Set the Facebook AppId and AppSecret as application setting in the Azure Web App portal. The configuration system is setup to read keys from environment variables.
+* Reference provider-specific pages to configure logins for the providers required by your app.
```

Comment:
Wont-fix for now. The heading is **next steps** indicating action, removing this bullet turns the section into a post-mortem summary. Should rename or remove whole section.